tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -12,32 +12,53 @@ parameters:
 - name: includeInternalVersions
   type: boolean
   default: false
+- name: buildToolsVersionToInstall
+  type: string
+  default: repo
 
 # Set version
 steps:
-- task: Bash@3
-  name: PrependPath
-  displayName: Prepend build-tools CLI to path
-  inputs:
-    targetType: 'inline'
-    workingDirectory: ${{ parameters.buildDirectory }}
-    script: |
-      # Prepend the cli bin dir to the path. See
-      # <https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#prependpath-prepend-a-path-to-the--path-environment-variable>
-      # more information.
-      echo "##vso[task.prependpath]$(Build.SourcesDirectory)/build-tools/packages/build-cli/bin"
+- ${{ if eq(parameters.buildToolsVersionToInstall, 'repo') }}:
+  - task: Bash@3
+    name: PrependPath
+    displayName: Prepend build-tools CLI to path
+    inputs:
+      targetType: 'inline'
+      workingDirectory: ${{ parameters.buildDirectory }}
+      script: |
+        # Prepend the cli bin dir to the path. See
+        # <https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#prependpath-prepend-a-path-to-the--path-environment-variable>
+        # more information.
+        echo "##vso[task.prependpath]$(Build.SourcesDirectory)/build-tools/packages/build-cli/bin"
+
+  - task: Bash@3
+    name: InstallBuildTools
+    displayName: Install Fluid Build Tools (from repo)
+    inputs:
+      targetType: 'inline'
+      workingDirectory: ${{ parameters.buildDirectory }}
+      script: |
+        pushd "$(Build.SourcesDirectory)/build-tools"
+        npm ci
+        popd
+
+- ${{ if ne(parameters.buildToolsVersionToInstall, 'repo') }}:
```

Comment:
Curious if we use an `if-equal` above and an independent `if-not-equal` here over an `else` for some reason?